INSTALL.md

```diff
@@ -0,0 +1,69 @@
+# Installation
+
+## Current development version
+Custom repositories and requirements must be set at root level during the current development phase.
+Edit the file, and add the following blocks:
+
+```json
+    "require": {
+        "composer/installers": "dev-ezplatform_assets as 1.3.x-dev",
+        "dpobel/ez-field-edit": "dev-master as 1.0.x-dev",
```

Comment:
Good catch @dpobel. It was indeed meant as a temporary workaround, the branch replaces it.